modules/ROOT/pages/use-anypoint-monitoring.adoc

```diff
@@ -0,0 +1,114 @@
+
+Each Mule app and API gateway replica can store up to 450 megabytes of log data on disk, after which log data 
+is rotated to provide storage for more recent log data.
+
+Logs generated by each replica of a Mule app and API gateway are collected and written to disk on the worker 
+node on which they are running. Each worker node runs a single log forwarding agent, which monitors this location 
+and transmits the log data to Anypoint Monitoring.
+
+The log forwarding agent can transmit up to 300 kilobytes of log data per second. You should not deploy more than
+50 replicas per worker node.
+
+In some cases, logs with multiple lines might not be fully transmitted to Anypoint Monitoring. This can occur if 
+multiple lines of log output are split across two files due to log rotation inside Runtime Fabric.
+
+You can enable log forwarding and set log levels when you xref:deploy-to-runtime-fabric.adoc#configure-logging-and-log-levels[deploy applications in Runtime Manager]. 
+
+=== Resource and Permission Requirements for Anypoint Monitoring Agent
+
+MuleSoft runs Anypoint Monitoring agents alongside Mule apps to send telemetry to the Anypoint control plane.
+
+For Mule apps deployed or managed using Runtime Fabric agent version 2.5.x or lower, two different agents are used as follows:
+
+* One DaemonSet per underlying node to transport logs to Anypoint monitoring control plane.
+* One sidecar per application replica to transporting metrics to Anypoint monitoring control plane. 
+
+For optimal daemonset functionality, follow these prerequisites:
+
+. Check that the application containers write logs to the following locations:
+
+* `/var/log/pods`
+* `/var/log/containers`
+* `/var/lib/docker/containers`
+
+. Check that the DaemonSet creates and writes on the host path `/var/lib/filebeat-data`.
+. Ensure that the Docker daemon config includes the `json-file` logging driver.
+. Ensure that the Anypoint Monitoring log forwarder DaemonSet container has write access to the host on the host path `/var/lib/filebeat-data`.
+
+For applications deployed or managed using Runtime Fabric versions 2.6.x and above, MuleSoft consolidates telemetry transport to a single sidecar per application replica.
+
+Dependent on the features you use, the sidecar agent consumes corresponding resources for optimal performance:
+
+To enable default Anypoint metrics, the sidecar agent (per application replica) requires 0.05vCPU or Core of CPU allocation and 90MiB memory allocation.
+
+If you opt into xref:runtime-fabric/deploy-to-runtime-fabric.adoc#configure-logging-and-log-levels[forwarding application logs to Anypoint Monitoring], the sidecar agent (per application replica) requires up to 30% of CPU limit configured for application replica and up to 150MiB memory allocation
+
+Allocate capacity to the underlying cluster by using maximum configurations to avoid application deployment failure.
```

Comment:
Note: Please allocate appropriate capacity to the underlying cluster to avoid application deployment failure.